src/main/webapp/i18n/de/lectureUnit.json

```
@@ -114,6 +114,41 @@
             "created": "Eine neue Dateieinheit wurde erstellt",
             "updated": "Dateieinheit aktualisiert",
             "notReleasedTooltip": "Nur sichtbar für Tutor:innen und Lehrende. Veröffentlichungsdatum:",
+            "createAttachmentUnits": {
+                "informativeText": {
+                    "firstLine": "Artemis sucht nach der Gliederung und unterteilt das PDF in Einheiten.",
+                    "secondLine": "Artemis extrahiert den Namen der Einheit in der Gliederung.",
+                    "note": "Hinweis: Sie haben die Möglichkeit, die Informationen zu bearbeiten, bevor Sie die angehängten Einheiten endgültig übermitteln."
+                },
+                "noUnitDetected": "Es wurde keine Einheit erkannt und konnte nicht automatisch geteilt werden. Bitte fügen Sie Informationen für jede Einheit hinzu!",
```

```suggestion
                "noUnitDetected": "Es wurde keine Einheit erkannt und konnte nicht automatisch geteilt werden. Bitte füge Informationen für jede Einheit hinzu!",
```
Artemis generally uses the informal version of ‘you’ in German.